composition-go/shim/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@wundergraph/composition-shim",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
```

Comment:
Do we want to release and publish this package?

Reply:
No, strictly for bundling it with the Go library, but we reference it in `generate.sh` that's why I gave it a `@wundergraph/*` name.

Reply:
ok, then you would need to make it a private package to prevent it from publishing.

Reply:
I didn't even know that was a thing! I've added it, PTAL.